src/content-helper/editor-sidebar/smart-linking/store.ts

```diff
@@ -81,12 +91,18 @@ interface SetWasAlreadyClickedAction {
 	wasAlreadyClicked: boolean;
 }
 
+interface SetApplyToAction {
```

Comment:
A comment explaining the interface would be nice 🙂

Reply:
The other interfaces in the store do not have any comments. But happy to add comments to this interface and the others if you think it's reasonable.

Reply:
Ah, this had escaped me. I'm always in favor of comments if they can give some clarity. You don't need to do anything at the moment, unless you want to.